src/libraries/System.Drawing.Primitives/src/System/Drawing/KnownColorNames.cs

```diff
@@ -149,6 +149,7 @@ internal static class KnownColorNames
             "Plum",
             "PowderBlue",
             "Purple",
+            "RebeccaPurple",
```

Comment:
Remove this line, this is duplicate, seems like forgot to remove it when moved to the end of the list.

Reply:
fixed thanks